cmd/chartsvc/handler.go

```diff
@@ -217,11 +217,17 @@ func newChartResponse(c *models.Chart) *apiResponse {
 }
 
 func newChartListResponse(charts []*models.Chart) apiListResponse {
+	chartDict := map[string]*apiResponse{}
```

Comment:
I would call this `chartDigests` and add a comment above this to describe that this is used to track chart versions by digests to ensure uniqueness